src/main/java/edu/kis/powp/jobs2d/drivers/CompositeDriver.java

```diff
@@ -0,0 +1,42 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CompositeDriver implements Job2dDriver {
+
+    private List<Job2dDriver> drivers;
+
+    public CompositeDriver(){
+        drivers = new ArrayList<>();
+    }
+
+    @Override
+    public void setPosition(int x, int y) {
+        drivers.forEach(e->e.setPosition(x, y));
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        drivers.forEach(e->e.operateTo(x, y));
+    }
+
+    public void add(Job2dDriver driver){
+        drivers.add(driver);
+    }
+
+    public List getChildren(){
```

Comment:
Why without a template? 
It is better to return a list copy so it cannot be modified from outside.